config/tsconfig.red.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "outDir": "../build",
+    "rootDir": "../src"
+  },
+  "include": [
```

Comment:
Why do we need a separate config for just RED?

Reply:
Same question. Using the existing tsconfig file I see `RedundantAudioEncoder.js` being generated in the build folder? Doesn't that mean we don't need this extra tsconfig file?

Reply:
I assume this is because we need to convert ts -> js for this file first and put it in the source folder so when we transpile for the whole folder they can refer to this file

Reply:
The idea for generating and building the worker code was:
1. Build the `RedundantAudioEncoder.js` file
2. Use the `RedundantAudioEncoder.js` file to generate `RedundantAudioEncoderWorkerCode.ts`
3. Now build everything using `npm run tsc:src` so that the generated `RedundantAudioEncoderWorkerCode.ts` file also gets built

So yes, we don't need this extra tsconfig file, but using the normal `tsconfig.json` would generate all of the built files for Step 1, then the `RedundantAudioEncoderWorkerCode.ts` file would be generated in Step 2, and then all of the files would be re-built again in Step 3. Having the separate config for RED was just a minor build time optimization so that we wouldn't be building all of the code twice during each `npm run build`.